lib/linguist/languages.yml

```diff
@@ -6375,6 +6375,18 @@ SuperCollider:
   tm_scope: source.supercollider
   ace_mode: text
   language_id: 361
+Synopsys Design Constrants:
+  type: programming
+  color: "#1D4994"
+  extensions:
+  - ".sdc"
+  - ".xdc"
+  filenames:
+  tm_scope: source.tcl
+  ace_mode: tcl
+  codemirror_mode: tcl
+  codemirror_mime_type: text/x-tcl
+  language_id: 96566567
```

Comment:
This is adding a whole new language which isn't necessary as the files are written in TCL. It's also going to clash with your addition of the same extensions to the TCL section.

It's best to do as requested in the original issue and you've stated in the title and OP and just add the extensions to TCL.

```suggestion
```

Reply:
Aw heck that should not be in there. I was experimenting with the new language addition and goofed. 🤦